resources/benchmark-runner.mjs

```diff
@@ -41,43 +41,59 @@ class BenchmarkState {
         return !this._testIndex;
     }
 
-    prepareCurrentSuite(runner, frame) {
+    async prepareCurrentSuite(frame, page) {
```

Comment:
The diff is still a bit messy due to depending on PR #19. 
For easier reviewing, I'll leave this PR as WIP until we land #19.